bin/pgut/pgut.c

```diff
@@ -467,13 +471,15 @@ prompt_for_password(void)
 	}
 #else
 	if (have_passwd) {
-		buf = pgut_malloc(BUFSIZE);
-		memcpy(buf, passwdbuf, sizeof(char)*BUFSIZE);
+		buf = pgut_malloc(passwd_size);
```

Comment:
`passwd_size` is equal to 0?

Reply:
Nope, `passwd_size` is static variable and the value is assigned here:
https://github.com/reorg/pg_repack/blob/905fd9f1931a3b78619ccb4eb25e57516ce7eaa8/bin/pgut/pgut.c#L478